packages/amplify-util-mock/src/utils/lambda/execute.ts

```diff
@@ -1,8 +1,8 @@
 import { existsSync } from 'fs-extra';
 import _ = require('lodash');
```

Comment:
could we change this to `import .. from ...` as well?

Reply:
ah whoops! Good catch